Documents/start_contribution.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to JDI Light
+
+## Step 1: Clone the Repository 
+
+First, you'll need to clone the repository to your local machine. You can do this with the following command: git clone https://github.com/jdi-testing/jdi-light.git
+
+## Step 2: Navigate to the Project Directory 
+
+Next, navigate to the project directory: cd jdi-light
+
+## Step 3: Understand the Properties Files 
+The JDI Light project uses *.properties files to configure various aspects of the project. These files are located in the src/test/resources directory. For example, the chrome.properties file is used to configure the Chrome driver, and the ff.properties file is used to configure the Firefox driver.
+
+The test.properties file is a general configuration file that contains various settings for the tests, such as the base URL for the tests, the driver to use, and other settings. Make sure to understand and update these files as necessary for your specific setup.
+
+for example to run tests in non-headless mode you need configure both common.properties and test.properties:
```

Comment:
We have one more option for parameters - specific for browser
https://github.com/jdi-testing/jdi-light/blob/master/jdi-light-html-tests/src/test/resources/chrome.properties

Reply:
added some info about chrome.properties